source/weekly-calendar.js

```diff
@@ -108,52 +64,33 @@ function init() {
     });
 }
 
-/**
- * Checks for tasks on a given date and adds them to the provided list element.
- * 
- * @function checkTasks
- * @param {HTMLElement} ulElt - The unordered list element to append tasks to.
- * @param {number} day - The day of the month.
- * @param {number} month - The month (0-11).
- * @param {number} year - The full year.
- */
-function checkTasks(ulElt, day, month, year) {
-    parser.getTasks().forEach((task) => {
-        const date = new Date(task["date"].replace(/-/g, '\/'));
+function checkTasks(day, month, year) {
+    const tasks = parser.getTasks();
+    let importance = 0;
 
-        if (
-            day === date.getDate() &&
-            month === date.getMonth() &&
-            year === date.getFullYear()
-        ) {
-            let eventCard = document.createElement("event-card");
-            eventCard.data = task;
-            ulElt.appendChild(eventCard);
+    tasks.forEach(task => {
+        const date = new Date(task["date"].replace(/-/g, '\/'));
+        if (day === date.getDate() && month === date.getMonth() && year === date.getFullYear()) {
+            importance += task["importance"];  // Assuming each task has an "importance" field
         }
     });
+
+    return importance;
 }
 
-/**
- * Generates the weekly calendar view for the specified day, month, and year.
- * 
- * @function makeCalendar
- * @param {number} day - The starting day of the week.
- * @param {number} month - The month (0-11).
- * @param {number} year - The full year.
- */
 function makeCalendar(day, month, year) {
     const monthTitle = document.getElementById("month-title");
     const week = [
-        document.getElementById("sunday"),
         document.getElementById("monday"),
         document.getElementById("tuesday"),
         document.getElementById("wednesday"),
         document.getElementById("thursday"),
         document.getElementById("friday"),
         document.getElementById("saturday"),
+        document.getElementById("sunday"),
```

Comment:
Why was sunday moved to the end of the list? Don't we want Sunday to be the first day of the week?